AGP-130: Aragon Mesh Sponsorship (#130)
aragon/AGPs: AGPs/AGP-120.md

---
AGP: 120

> Title: 

Reviewer: AGP needs a title

Author: Aragon Mesh Team (@aragonmesh on keybase), Gabriel Garcia (@0xGabi), Daniel Constantin (@0x6431346e)
Status: Stage IV
Track: Finance
Created: 2019-11-18
---

### Summary

Aragon Mesh started as a [Nest project](https://github.com/aragon/nest/pull/151) to fund the development of the aragonCLI. To manage the grant funds we implemented a milestone-based process inspired by [Giveth](https://giveth.io), where contributors would summarize their work over a given period alongside a reasonable pay rate and the other DAO members would vote whether to approve their compensation. We think this process is low-overhead and fits the DAO very well at this stage.

Our mission is to improve the developer experience as a whole, which includes easing the on-boarding process of new developers that wish to work on the Aragon core infrastructure & SDK. This AGP aims to compensate those devs.

> In this new endeavor we are looking forward to formalizing and improving the current process. We have been keeping track of milestones in this GitHub repository: [AragonMesh/team](https://github.com/AragonMesh/team). As part of improving the current process we will like to continue experimenting with the [Projects app](https://github.com/AutarkLabs/open-enterprise/blob/dev/docs/GETTING_STARTED.md#install-the-projects-app) of the Open Enterprise suite and bounty-based compensation.

Reviewer: ```suggestion
In this new endeavor we are looking forward to formalizing and improving the current process. We have been keeping track of milestones in this GitHub repository: [AragonMesh/team](https://github.com/AragonMesh/team). As part of improving the current process we would like to continue experimenting with the [Projects app](https://github.com/AutarkLabs/open-enterprise/blob/dev/docs/GETTING_STARTED.md#install-the-projects-app) of the Open Enterprise suite and bounty-based compensation.
```

## Address of the transfer recipient

Mesh Mainnet DAO Vault Address: `0xe549556e796d04713812cffa2b457bc7b5d3366f`

## Amount of the transfers

30,000 DAI

5,000 ANT

## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)

> Quarterly transfers of 30,000 DAI minus unused funds from the previous quarter if any. This is the amount to be debited from the quarterly Finance track budget.

Reviewer: Is this meant to indicate that only the DAI portion of the specified transfers is to be recurring quarterly?

Author: Good catch, I added ANT as well to recurring transfer.

The transfer will be made after each ANV has completed.

## Purpose of the transfer

> This sponsorship is to keep a continue stream of funding to support new developers wanting to contribute to core infrastructure of Aragon. ANT will be used as [reward](https://www.notion.so/Reward-system-ce1b3893f8a3496b801d88675f9f7250) mechanism.

Reviewer: ```suggestion
This sponsorship is to keep a continue stream of funding to support new developers wanting to contribute to core infrastructure of Aragon. ANT will be used as a [reward](https://www.notion.so/Reward-system-ce1b3893f8a3496b801d88675f9f7250) mechanism.
```

> The founds will be controller by token holders of the `MESH` token. 

Reviewer: ```suggestion
The funds will be controlled by token holders of the `MESH` token. 
```
